app/views/medications/edit.html.erb

```diff
@@ -1,2 +1,4 @@
 <% title "#{t('common.actions.edit_instance', instance: @medication.name)}" %>
+
+<%= render 'validation_errors.html.erb' %>
```

Comment:
Can this just be `<%= render 'validation_errors' %>`?

Reply:
It most certainly! Good catch!